cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -88,8 +87,7 @@ public void init() {
         s3Client = new FakeS3Client(new ConcurrentHashMap<>());
         handler = new CristinEntryEventConsumer(resourceService, s3Client);
         outputStream = new ByteArrayOutputStream();
-        cristinDataGenerator = new CristinDataGenerator();
-        super.testingData = cristinDataGenerator.singleRandomObjectAsString();
+        super.testingData = CristinDataGenerator.singleRandomObjectAsString();
```

Comment:
cristinDataGenerator has now only static methods and there is no need for a  specific object